src/interactive/__snapshots__/accordionPanel.test.jsx.snap

```diff
@@ -172,7 +172,7 @@ exports[`AccordionPanel Panel with right aligned icon exists and renders icon le
             <FlexItem
               id="label-firstsection"
             >
-              <FlexItem
+              <FlexItemComponent
```

Comment:
@mmcgahan - Some snapshots are rendering components wrapped in `withLoading` with `NameComponent` wrapped in `Component`.
Do you know why this happens? Is this something we need to change or "fix"?

Reply:
that's expected - since `withLoading` adds a wrapper around the base component, `mount`-rendered snapshots will show the 'outer' layer with the expected component name as well as the 'inner' component with the real component's name. We should generally use `shallow` for snapshots